components/popover/positionCalculation.js

```diff
@@ -88,11 +88,11 @@ const getHorizontalDirectionPositionTopValue = ({
   inputOffsetCorrection,
 }) => {
   switch (position) {
-    case 'middle':
-      return anchorPosition.middle - popoverDimensions.height / 2;
-    case 'top':
+    case 'center':
```

Comment:
PP: I would work with constants for position and define these somewhere so that is clear what options are possible.

Reply:
@JorenSaey 	would you suggest something like this?

````
const POSITION_START = 'start';
const POSITION_CENTER = 'center';
const POSITION_END = 'end';
````